sys-cluster/charliecloud/charliecloud-0.9.10.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/hpc/${PN}.git"
+	S="${WORKDIR}/${P}"
+else
+	SRC_URI="https://github.com/hpc/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="Lightweight user-defined software stacks for high-performance computing"
+HOMEPAGE="https://hpc.github.io/charliecloud/"
+
+SLOT="0"
+LICENSE="Apache-2.0"
+IUSE="doc examples +pv"
+
+RDEPEND=""
+DEPEND="${RDEPEND}
+	pv? ( sys-apps/pv )
```

Comment:
How is this a build-time dependency? It looks weird, given that `pv` flag is not used anywhere else. Does it enable some progress bars during build, or is it actually a runtime dependency?

Also, we generally try to keep flags sorted.

Reply:
That's a runtime dependency, of course! I wonder how I missed that, sorry and thanks for pointing this out!